Stanford/JumpstartEngineering/Install/Layouts/DeptHomepage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
+ * Configure PICAL homepage layouts.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class DeptHomepage extends \AbstractInstallTask {
+
+  /**
+   * Configure Department homepage layouts.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
- Dont mess with the other contexts
- Save your new context to the DB
- Enable the context
- Disable the currently active homepage context so your new context is the truth